README.md

```diff
@@ -103,3 +103,20 @@ These Kotlin classes are supported with the following fields for serialization/d
 * LongRange _(start, end)_
 
 (others are likely to work, but may not be tuned for Jackson)
+
+# Configuration
+
+The Kotlin module may be given a few configuration parameters at construction time; see the [inline documentation](https://github.com/FasterXML/jackson-module-kotlin/blob/master/src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt) for details on what options are available and what they do.
+
+```kotlin
+val mapper = ObjectMapper().registerModule(KotlinModule(strictNullChecks = true))
```

Comment:
May want to expose new (2.10+) builder style, alternatively:

```
val mapper = JsonMapper.builder().addModule(...).build();
```

Reply:
Good idea, updated.